tests/test_interface_private_vlan.rb

```diff
@@ -321,6 +321,14 @@ def test_switchport_pvlan_trunk_allowed_vlan
 
     i.switchport_pvlan_trunk_allowed_vlan = default
     assert_equal(default, i.switchport_pvlan_trunk_allowed_vlan)
+
+    vlans = '500-528,530,532,534,587,590-593'
+    all_vlans = '500-528,530,532,534,587,590-593,597-598'
+    config("interface #{i.name}",
+           "switchport private-vlan trunk allowed vlan #{vlans}",
```

Comment:
Does this test result in multiple add lines on all platforms?  If not please make sure to modify it so that it does.